src/App.tsx

```diff
@@ -1,56 +1,99 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
+import { debounce } from 'lodash';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { PeopleList } from './components/PeopleList';
```

Comment:
<img width="661" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/70220667/6a03313a-74da-4c43-8d5f-3626019372d0">
you forgot to add this component